[Backport release/1.2] STP-3192: organization and linting of FAS topics (#1460)
Cray-HPE/docs-csm: operations/firmware/FAS_CLI.md

The following CLI operations are described:

1. Action
   1. [Execute an Action](#action)
   2. [Abort an Action](#abort)
   3. [Describe an Action](#describe)
2. Snapshots
   1. [Create a Snapshot](#create)
   2. [View a Snapshot](#view)
   3. [List Snapshots](#list)
3. [Update an Image](#update)
4. [FAS Loader Commands](#loader)

## Prerequisites

The Cray command line interface (CLI) tool is initialized and configured on the system.

<a name="action"></a>

## Execute an Action

> Use FAS to execute an action. An action produces a set of firmware operations. Each operation represents a component name (xname) + target on that component name (xname) that will be targeted for update. There are two of firmware action modes: : `dryrun` or `liveupdate`; the parameters used when creating either are completely identical except for the `overrideDryrun` setting. `overrideDryrun` will determine if feature to determine what firmware can be updated on the system. Dry-runs are enabled by default, and can be configured with the `overrideDryrun` parameter. A dry-run will create a query according to the filters requested by the admin. It will initiate an update sequence to determine what firmware is available, but will not actually change the state of the firmware

Reviewer: ```suggestion
Use FAS to execute an action. An action produces a set of firmware operations. Each operation represents a component name (xname) and a target on that xname that will be targeted for update. There are two modes of firmware action: `dryrun` and `liveupdate`. The parameters used when creating either are identical except for the `overrideDryrun` setting. The `overrideDryrun` setting is `false` by default. This mode (the dry-run mode) will create a query according to the filters requested by the administrator. This will initiate an update sequence to determine what firmware is available, but will not actually change the state of the firmware.
```

```

> The loader can only run one job at a time, if the loader is `busy`, it will return an error on any attempt to create an additional job.

Reviewer: ```suggestion
The loader can only run one job at a time; if the loader is `busy`, it will return an error on any attempt to create an additional job.
```
